tests/Unit/Domain/CoordinateMaps/TimeDependent/Test_Skew.cpp

```diff
@@ -0,0 +1,123 @@
+  std::uniform_real_distribution<double> outer_radius_dist{50.0, 150.0};
+  std::uniform_real_distribution<double> point_dist{-5.0, 5.0};
+  // NOLINTEND
+
+  using Polynomial = domain::FunctionsOfTime::PiecewisePolynomial<deriv_order>;
+  using FoftPtr = std::unique_ptr<domain::FunctionsOfTime::FunctionOfTime>;
+  std::unordered_map<std::string, FoftPtr> f_of_t_list{};
+  f_of_t_list[function_of_time_name] = std::make_unique<Polynomial>(
+      initial_time,
+      std::array{make_with_random_values<DataVector>(
+                     generator, make_not_null(&fot_dist), DataVector{2, 0.0}),
+                 make_with_random_values<DataVector>(
+                     generator, make_not_null(&fot_dist), DataVector{2, 0.0}),
+                 DataVector{2, 0.0}},
+      expiration_time);
+
+  const std::array<double, 3> center{
+      point_dist(*generator), point_dist(*generator), point_dist(*generator)};
+  const double outer_radius = outer_radius_dist(*generator);
+  CAPTURE(center);
+  CAPTURE(outer_radius);
+
+  const CoordinateMaps::TimeDependent::Skew skew_map{function_of_time_name,
+                                                     center, outer_radius};
+
+  CHECK(skew_map.function_of_time_names().contains(function_of_time_name));
+
+  // test serialized/deserialized map
+  const auto skew_map_deserialized = serialize_and_deserialize(skew_map);
+  CHECK(skew_map_deserialized.function_of_time_names().contains(
+      function_of_time_name));
+
+  while (t < expiration_time) {
+    CAPTURE(t);
+    const std::array<double, 3> point_xi{
+        point_dist(*generator), point_dist(*generator), point_dist(*generator)};
+
+    const std::array<DataVector, 3> dv_point_xi{
+        make_with_random_values<DataVector>(
+            generator, make_not_null(&point_dist), DataVector{10, 0.0}),
+        make_with_random_values<DataVector>(
+            generator, make_not_null(&point_dist), DataVector{10, 0.0}),
+        make_with_random_values<DataVector>(
+            generator, make_not_null(&point_dist), DataVector{10, 0.0})};
+
+    const auto run_checks = [&](const auto& points) {
+      CAPTURE(points);
+      test_jacobian(skew_map, points, t, f_of_t_list);
+      test_inv_jacobian(skew_map, points, t, f_of_t_list);
+      test_frame_velocity(skew_map, points, t, f_of_t_list);
+
+      test_jacobian(skew_map_deserialized, points, t, f_of_t_list);
+      test_inv_jacobian(skew_map_deserialized, points, t, f_of_t_list);
+      test_frame_velocity(skew_map_deserialized, points, t, f_of_t_list);
+    };
+
+    run_checks(point_xi);
+    test_coordinate_map_argument_types(skew_map, point_xi, t, f_of_t_list);
+    test_coordinate_map_argument_types(skew_map_deserialized, point_xi, t,
+                                       f_of_t_list);
+    test_inverse_map(skew_map, point_xi, t, f_of_t_list);
+    test_inverse_map(skew_map_deserialized, point_xi, t, f_of_t_list);
+    run_checks(dv_point_xi);
```

Comment:
Maybe check that the map gives expected results at some points that you know, like at the center and at the outer radius